frontend/shared/forms/renderField.js

```diff
@@ -124,13 +124,13 @@ const renderRadio = ({ input, className, label, options, hideLabel, meta: { touc
                     checked={input.value === option.value}
                     value={option.value}
                 />
-                <label className="form-check-label" htmlFor={`${input.name}${index}`}>
+                <span className="form-check-label" htmlFor={`${input.name}${index}`}>
```

Comment:
this should be changed backed to `label` as a lot of people are used to clicking labels to select a radio button